src/pretalx/agenda/management/commands/export_schedule_html.py

```diff
@@ -1,74 +1,230 @@
+            for url in (get_path(urllib.parse.unquote(url)) for url in css_assets):
+                logging.debug(url)
+                content = get_mediastatic_content(url)
+                dump_content(dest, url, content)
+
+
+def rm_rf(path):
+    try:
+        shutil.rmtree(path)
+    except FileNotFoundError:
+        pass
+
+
+def get_export_path(event):
+    return settings.HTMLEXPORT_ROOT / event.slug
+
+
+def get_export_zip_path(event):
+    return str(get_export_path(event)) + '.zip'
+
+
+class Command(BaseCommand):
 
     def add_arguments(self, parser):
         super().add_arguments(parser)
         parser.add_argument('event', type=str)
         parser.add_argument('--zip', action='store_true')
 
-    @classmethod
-    def get_output_dir(cls, event):
-        # Do not change, this is used to build the correct zip path
-        return settings.HTMLEXPORT_ROOT / event.slug
-
-    @classmethod
-    def get_output_zip_path(cls, event):
-        return Path(str(cls.get_output_dir(event)) + '.zip')
-
-    def build_media(self):
-        output_dir = self.get_output_dir(self._exporting_event)
-        (output_dir / 'media' / self._exporting_event.slug).mkdir(parents=True, exist_ok=True)
-        return super().build_media()
-
     def handle(self, *args, **options):
         event_slug = options.get('event')
+
         try:
             event = Event.objects.get(slug__iexact=event_slug)
         except Event.DoesNotExist:
             raise CommandError(f'Could not find event with slug "{event_slug}".')
 
-        self._exporting_event = event
-        translation.activate(event.locale)
-
-        output_dir = self.get_output_dir(event)
-        with override_settings(
-            COMPRESS_ENABLED=True,
-            COMPRESS_OFFLINE=True,
-            BUILD_DIR=str(output_dir),
-            MEDIA_URL=str(Path(settings.MEDIA_URL) / event_slug),
-            MEDIA_ROOT=str(Path(settings.MEDIA_ROOT) / event_slug),
-        ):
-            with override_timezone(event.timezone):
-                super().handle(*args, **options)
-                if options.get('zip', False):
-                    make_archive(
-                        base_name=output_dir,
-                        format='zip',
-                        root_dir=settings.HTMLEXPORT_ROOT,
-                        base_dir=event.slug,
-                    )
-                    output_dir = self.get_output_zip_path(event)
-        self.stdout.write(str(output_dir))
-
-    def build_views(self):
-        for view_str in self.view_list:
-            view = get_callable(view_str)
-            view(_exporting_event=self._exporting_event).build_method()
+        export_dir = get_export_path(event)
+        tmp_dir = str(export_dir) + '-new'
+
+        rm_rf(tmp_dir)
+        os.mkdir(tmp_dir)
+
+        try:
+            export_event(event, tmp_dir)
+            rm_rf(export_dir)
+            os.rename(tmp_dir, export_dir)
```

Comment:
```
tmp_dir.rename(export_dir)
tmp_dir = export_dir
```

(Yeah, pathlib renaming is super weird.)

Reply:
doesn't `tmp_dir = export_dir` remove `htmlexport/dj19` instead of `htmlexport/dj19-new` in the next line?

Reply:
Yeah, ignore the assignment suggestion if you never access `tmp_dir` again.